js/whisper-demo/index.js

```diff
@@ -34,7 +34,13 @@ function updateConfig() {
             options.dataType = pair[1];
         }
         if (pair[0] == 'maxChunkLength') {
-            options.maxChunkLength = parseFloat(pair[1]);
+            options.maxChunkLength = parseInt(pair[1]);

```

Comment:
I'd like to keep this as `ParseFloat`, as we may try to reduce the `maxChunkLength` within 1 sec to get better real-time experience once we get better performance optimization.

Reply:
Makes sense

Reply:
Reverted in latest commit.